src/panels/lovelace/common/create-row-element.js

```diff
@@ -12,7 +12,7 @@ import "../entity-rows/hui-scene-entity-row.js";
 import "../entity-rows/hui-script-entity-row.js";
 import "../entity-rows/hui-text-entity-row.js";
 import "../entity-rows/hui-timer-entity-row.js";
-import "../entity-rows/hui-toggle-entity-row.js";
+import "../entity-rows/hui-toggle-entity-row";
```

Comment:
We should just have a PR that rips out all `.js` from all imports throughout the whole repo.

Reply:
Just remember that you asked for this: https://github.com/home-assistant/home-assistant-polymer/pull/1941
I started small with only all of `src` 😄